spec/models/author_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Author, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'method checking' do
+    it 'if data stores' do
+      author = Author.new(id: 1, name: 'abc', email: 'abc@gmail.com')
+      expect(author).to be true
+    end
+
+    it 'if data first receives' do
+      Author.new(id: 1, name: 'pqr', email: 'pqr@gmail.com')
+      author = Author.first
```

Comment:
If you are not saving anything to database how come `Author.first` fetches any record.